frontend/views/components/group-creation-steps/GroupRules.vue

```diff
@@ -130,9 +130,6 @@ export default {
              this.group.memberApprovalThreshold >= SUPERMAJORITY &&
              this.group.memberRemovalThreshold >= SUPERMAJORITY
     }
-  },
-  mounted () {
-    this.$emit('focusref', 'finish')
```

Comment:
> Also, I wasn't able to test this because when I got to the final step of the "Create Group" flow I got the following error in Firefox Dev Edition:
>> uncaught Vue error in v-on handler: TypeError: "this.$refs[ref].focus is not a function"
    focusRef GroupCreationModal.vue:96

@taoeffect, I also see the error but it didn't prevent me from creating the group as normal.

These lines were the cause: it tried to focus the submit button once the final step is visible, but that's a bad practice because it's messing with the default keyboard navigation (here are other important interactive elements before the submit button).

A more correct solution would be to instead focus on the beginning of the step each time we move between steps.

Anyway, that's a problem for another PR. For now, I just removed the forced focus.

Reply:
Hmm, that explanation does not make sense to me. This exception does not happen in `master`, and yet these lines are present in `master`.

The problem has to be something with this PR.

Reply:
These lines are calling the ref "finish" at `GroupCreationModal.vue` that is the submit button. But now the button is a component `ButtonSubmit`, not an element `<button />`, so you cannot focus a component. But as I said, focusing the submit button here is a bad practice. So, I removed the code responsible for triggering the button focus.

Reply:
> so you cannot focus a component

Ah I see. Being able to focus a component though seems like a potentially legitimate use-case, can you check to see if there's a way to make this work anyway (even if you feel that it shouldn't be focused in this case)? I.e. is there a way to add the `focus` method to the component so that an exception isn't thrown when it's used like this?

Reply:
In other words, I don't think we should lose the ability to focus buttons just because we're using `ButtonSubmit.vue` in some places.

Reply:
Yes, ofc it's possible. Added it back and did the needed change to `ButtonSubmit.vue` :)